src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -193,28 +215,32 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
             readonly={true}
           />
         </Grid>
-        <Grid padding={theme.spacing(1, 3, 1, 5)} xs={12}>
-          <Select
-            label={strings.SUBSTRATE}
-            placeholder={strings.SELECT}
-            options={getSubstratesAccordingToType(record?.testType)}
-            onChange={(value: string) => onChange('substrate', value)}
-            selectedValue={record?.substrate}
-            fullWidth={true}
-            readonly={true}
-          />
-        </Grid>
-        <Grid padding={theme.spacing(1, 3, 1, 5)} xs={12}>
-          <Select
-            label={strings.TREATMENT}
-            placeholder={strings.SELECT}
-            options={TREATMENTS}
-            onChange={(value: string) => onChange('treatment', value)}
-            selectedValue={record?.treatment}
-            fullWidth={true}
-            readonly={true}
-          />
-        </Grid>
+        {record?.testType !== 'Cut' && (
+          <>
+            <Grid padding={theme.spacing(1, 3, 1, 5)} xs={12}>
+              <Select
+                label={strings.SUBSTRATE}
+                placeholder={strings.SELECT}
+                options={getSubstratesAccordingToType(record?.testType)}
+                onChange={(value: string) => onChange('substrate', value)}
+                selectedValue={record?.substrate}
+                fullWidth={true}
+                readonly={true}
+              />
+            </Grid>
+            <Grid padding={theme.spacing(1, 3, 1, 5)} xs={12}>
+              <Select
+                label={strings.TREATMENT}
+                placeholder={strings.SELECT}
+                options={TREATMENTS}
+                onChange={(value: string) => onChange('treatment', value)}
+                selectedValue={record?.treatment}
+                fullWidth={true}
+                readonly={true}
+              />
```

Comment:
can you rebase? I adde the `disabled` prop on all these fields (for completed tests)